a3p-integration/proposals/a:upgrade-next/package.json

```diff
@@ -18,7 +16,10 @@
   },
   "ava": {
     "concurrency": 1,
-    "serial": true
+    "serial": true,
+    "files": [
+      "!submission"
+    ]
```

Comment:
I suspect this isn't necessary. This proposal package doesn't have a `submission` directory. `b:local-chain` does, and it has a `test-localchain.js` which Ava in most agoric-sdk packages would match because they override Ava's default. But I think the default glob is `*.test.js` so it wouldn't pick up `test-*` here.

Not a blocker because I'm not certain and about to work in this code after you land this so I can clean it up (after verifying).

Reply:
Ah, it must be inheriting the config from the SDK root package.json. I will address later